### PR TITLE
Apache2 module integration test

### PR DESCRIPTION
**Pull request by robinro**
##### ISSUE TYPE
- Feature Pull Request
##### ANSIBLE VERSION

devel
##### SUMMARY

Add integration test for apache2_module.

Tests (so far):
- whether the module works at all
- idempotency
- implicit: test it works on the distributions used for integration tests

### CODE REVIEW

File: test/integration/roles/test_apache2_module/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+
+- name: include only on supported systems
+  include: actualtest.yml
+  when: ansible_os_family in ['Debian', 'Suse']
```
Comment (gundalow) [inline test/integration/roles/test_apache2_module/tasks/main.yml:5]: Why doesn't this work on all RHEL based OS?

Comment (robinro) [inline test/integration/roles/test_apache2_module/tasks/main.yml:5]: RHEL (at least centos7, which I tested) does not have a2enmod/a2dismod

As far as I understand debian came up with it and opensuse also implemented it, but independently and with other arguments.

**Comment by robinro** [comment]
@gundalow as you already had a look, can this be merged?

**Comment by robinro** [comment]
thanks @resmo for the merge

**Pull request merged**